src/js_native_api_v8.h

```diff
@@ -1,7 +1,6 @@
 #ifndef SRC_JS_NATIVE_API_V8_H_
 #define SRC_JS_NATIVE_API_V8_H_
 
-// This file needs to be compatible with C compilers.
```

Comment:
This header file contains c++ syntaxes. I don't think this comment is meaningful.

Reply:
I think that would also apply to the lint rule in the line below this one, if you want to address that.

Reply:
Thanks for pointing that out. Removed the NOLINT mark and moved the include directive to `node_api.cc`, where the functions defined in header `cstring` are actually used.